### DISCUSSION TITLE
So.. When will chart submission will be public to upload on client side?

### DISCUSSION THREAD

**Discussion by maiz94** [TeamFlos/phira#245] [General]
Or still closed for _selected_ talented, experienced charters only for now?

**Comment by sjfhsjfh** [comment]
Chart -> Edit -> Upload?

**Reply by maiz94** [reply]
![Screenshot_2024-01-03-18-58-18-316_org flos phira](https://github.com/TeamFlos/phira/assets/7908304/286308b6-4864-4cdc-874d-ae37cba879e1)
I don't see any chart -> edit -> upload on there?

**Reply by sjfhsjfh** [reply]
Choose the chart using the Import button, then the click edit(the pencil icon)

Chart designing software is not provided, you may join [Phigros fanmade Discord server](https://discord.gg/U5DRNGk8qV) or [Discord Phira Community](https://discord.gg/MC92fRrsds) for further help

**Reply by maiz94** [reply]
I see. It's kind of vague that I didn't see an upload chart button on the song selection menu. Thanks for the help.

**Comment by maiz94** [comment]
> Choose the chart using the Import button, then the click edit(the pencil icon)
> 
> Chart designing software is not provided, you may join [Phigros fanmade Discord server](https://discord.gg/U5DRNGk8qV) or [Discord Phira Community](https://discord.gg/MC92fRrsds) for further help
